packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksInput.tsx

```diff
@@ -37,7 +37,13 @@ const BlocksInput = React.forwardRef<{ focus: () => void }, BlocksInputProps>(
       <>
         <Flex direction="column" alignItems="stretch" gap={1}>
           <Flex gap={1}>
-            <Typography variant="pi" fontWeight="bold" textColor="neutral800">
+            <Typography
+              variant="pi"
+              fontWeight="bold"
+              textColor="neutral800"
+              as="label"
+              id={`blocks-${label}`}
```

Comment:
This won't work if someone has the same label, it's also not the best practice for Ids in react. Please use `React.useId`.

In my other comment I asked about using `aria-labelledby` im thinking `htmlFor` is more appropriate though.